proxy/certs/certs.go

```diff
@@ -160,10 +160,17 @@ const (
 	backoffRetries = 5
 )
 
-func backoffAPIRetry(desc, instance string, do func() error) error {
-	var err error
+var now = func() time.Time {
+	return time.Now().UTC()
+}
```

Comment:
Is this function really needed? It seems like just a bit of indirection

Reply:
It looks like we are using this for testing. Can we add a comment here to clarify that?

Reply:
Will do.